src/device.coffee

```diff
@@ -40,8 +39,16 @@ exports.getID = memoizePromise ->
 			throw new Error('Could not find this device?!')
 		return devices[0].id
 
+waitForReboot = ->
+	console.log('Waiting for reboot...')
+	Promise.delay(3000).then(waitForReboot)
```

Comment:
You're right, I'll close this while I think of a better solution (otherwise it'll get stale quickly)